### ISSUE TITLE
Error in handling escape when color picker is open.

### ISSUE THREAD

**Issue by barklund** [GoogleForCreators/web-stories-wp#539]
## Bug Description

There are multiple things to _escape out of_ when opening the color picker for a given element or page background.

This should be handled correctly. Compare expected behavior with steps to reproduce below.

#### Element
1. Select a text element (or another element than can have color assigned)
2. Open the color picker for text color
3. Click on hex color to make it editable
4. Pressing <kbd>esc</kbd> once should exit edit mode for hex, but keep color picker open and element selected
5. Pressing <kbd>esc</kbd> again should close color picker, but keep element selected
6. Pressing <kbd>esc</kbd> again should unselect element

#### Page
1. Click the page background
2. Open the color picker for page background color
3. Click on hex color to make it editable
4. Pressing <kbd>esc</kbd> once should exit edit mode for hex, but keep color picker open and element selected
5. Pressing <kbd>esc</kbd> again should close color picker.

#### Element
1. Select a text element (or another element than can have color assigned)
2. Open the color picker for text color
3. Click on hex color to make it editable
4. Pressing <kbd>esc</kbd> once does exit edit mode and keeps color picker open and element selected
5. Pressing <kbd>esc</kbd> again **unselects element**
#### Page
1. Click the page background
2. Open the color picker for page background color
3. Click on hex color to make it editable
4. Pressing <kbd>esc</kbd> once does exit edit mode and keeps color picker open
5. Pressing <kbd>esc</kbd> again **does nothing**

## Screenshots

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
(just perusing the backlog, pretty sure this is resolved @miina / @barklund   if you wanna double check)

**Comment by miina** [comment]
@BrittanyIRL Testing with a shape, it looks like it's not working entirely correctly.
Steps:
1. Add a shape
2. Open the color picker
3. Click "Custom"
4. Focus on the hex input
5. Click "Esc" -> hex input gets unfocused (correct)
6. Click "Esc" again -> color picker gets closed AND the element gets unselected (Incorrect! The color picker should close but element stay selected).

(Feel free to take the issue if you'd like)

**Comment by BrittanyIRL** [comment]
We're good for now, just noticed it's age and thought maybe it was resolved based on what i tried.
